<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newsWindow</class>
 <widget class="QDialog" name="newsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>News Window</string>
  </property>
  <widget class="QComboBox" name="newsComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CTV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CBC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FOX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CNN</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="newsButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="youtubeCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Youtube</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="websiteCheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Website</string>
   </property>
  </widget>
  <widget class="QLabel" name="newsPicture">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>11</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="newsSourcePic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="newsBackButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>13</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>newsPicture</zorder>
  <zorder>newsComboBox</zorder>
  <zorder>newsButton</zorder>
  <zorder>newsSourcePic</zorder>
  <zorder>newsBackButton</zorder>
  <zorder>label1</zorder>
  <zorder>youtubeCheckBox</zorder>
  <zorder>label2</zorder>
  <zorder>websiteCheckBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
